iothub/device/src/InternalClient.cs

```diff
@@ -200,6 +199,7 @@ internal class InternalClient : IDisposable
             if (Logging.IsEnabled)
                 Logging.Exit(this, transportSettings, pipelineBuilder, nameof(InternalClient) + "_ctor");
         }
+        internal bool IsDisposed { get; private set; }
```

Comment:
Do you think we could make this public or would that go against IDisposable pattern (given that .NET doesn't expose such a property on IDisposable types)?

Reply:
I'm thinking about doing it for v2 but I don't know about adding new stuff to v1.